packages/config/config/devices/0x0455/ora-zrx.json

```diff
@@ -0,0 +1,203 @@
+			"label": "Follow Me",
+			"description": "The device in this group level",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Scene Activation",
+			"description": "Receive an activation scene ID if an obstacle are reach during its operation. The scene ID can be define by using parameters 30, 31",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Follow Me Weather",
+			"description": "The device in this group will follow the device level if the change is due by the weather condition. The parameter ID sent by this command can be configured by using parameter 32",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "30",
+			"label": "Open Collision Scene",
+			"description": "Scene ID sent if a collision is detected during opening operation",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 254,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Do not send the scene activation",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "31",
+			"label": "Close Collision Scene",
+			"description": "Scene ID sent if a collision is detected during closing operation",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 254,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Do not send the scene activation",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "32",
+			"label": "Parameter for Window Covering",
+			"description": "Parameter ID used by the window covering set on group Follow Me Weather",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 23,
+			"defaultValue": 13,
+			"unsigned": true
+		},
+		{
+			"#": "33",
+			"label": "Movement Trigger",
+			"description": "Parameter used to tell the reason of the last movement",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": true,
+			"options": [
+				{
+					"label": "Requested by user",
+					"value": 0
+				},
+				{
+					"label": "Wind",
+					"value": 1
+				},
+				{
+					"label": "Sun",
+					"value": 2
+				},
+				{
+					"label": "Rain",
+					"value": 3
+				}
+			]
+		},
+		{
+			"#": "34",
+			"label": "Automation Configuration",
+			"description": "The value of this parameter is the sum of wind, sun and rain status. Since automation for wind and rain are always activated the only valid values are: 5 to disable the sun automation or 7 to enabled it",
+			"valueSize": 1,
+			"minValue": 5,
+			"maxValue": 7,
+			"defaultValue": 5,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable sun automation",
+					"value": 5
+				},
+				{
+					"label": "Enable sun automation",
+					"value": 7
+				}
+			]
+		},
+		{
+			"#": "35",
+			"label": "Sensor Activation Status",
+			"description": "Allows to know which sensor has been associated to the motor. The value of this parameter is the sum of wind, sun and rain status",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 63,
```

Comment:
This looks like it should be split into 3 separate readonly partial parameter, one each for wind, sun and rain status. Does the manual say which bit is which sensor?

Reply:
Yes, manual is a little bit more verbose on that but I didn't understand it quite well plus didn't know how to put the information. Any suggestion/explanation is welcome :-)

![image](https://user-images.githubusercontent.com/75622100/163181375-85b315ad-9c67-4477-adef-cc1c12b59232.png)

Reply:
Interesting... assuming the values use the same bits (1+2+4) as in parameter 34, I'm not sure how this is going to go up to 63. Which value does it show on your device?

Reply:
My #35 is 0 but I'm afraid I don't have any sun/wind/rain sensor/device, only the motor itself

![image](https://user-images.githubusercontent.com/75622100/163185172-449d870f-a159-408e-a92c-6066c11b21f2.png)

Reply:
Ok, we'll leave this as is until someone comes who knows more.